packages/contracts-bedrock/contracts/test/BenchmarkTest.t.sol

```diff
@@ -0,0 +1,89 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+/* Testing utilities */
+import "./CommonTest.t.sol";
+import { CrossDomainMessenger } from "../universal/CrossDomainMessenger.sol";
+
+
+
+contract L1CrossDomainMessenger_GasBenchMark is Messenger_Initializer {
+    // The amount of data typically sent during a bridge deposit.
+
+    function test_L1MessengerSendMessage_benchmark() external {
+        bytes memory data = hex"ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
+        L1Messenger.sendMessage(bob, data, uint32(100));
+    }
+}
+
+contract L1StandardBridge_Deposit_GasBenchMark is Bridge_Initializer {
+    function setUp() public virtual override {
+        super.setUp();
+        deal(address(L1Token), alice, 100000, true);
+        vm.startPrank(alice, alice);
+    }
+
+    function test_depositETH_benchmark() external {
+        L1Bridge.depositETH{ value: 500 }(50000, hex"ff");
+    }
+
+    function test_depositERC20_benchmark() external {
+        L1Bridge.depositETH{ value: 500 }(50000, hex"ff");
+    }
+}
+
+contract L1StandardBridge_Finalize_GasBenchMark is Bridge_Initializer {
+    function setUp() public virtual override {
+        super.setUp();
+        deal(address(L1Token), address(L1Bridge), 100, true);
+        vm.mockCall(
+            address(L1Bridge.messenger()),
+            abi.encodeWithSelector(CrossDomainMessenger.xDomainMessageSender.selector),
+            abi.encode(address(L1Bridge.otherBridge()))
+        );
+        vm.startPrank(address(L1Bridge.messenger()));
```

Comment:
IIRC that didn't work for me, but I'll check again.

Reply:
Believe it or not, it doesn't work. Needs to be `startPrank`. If I had to guess I'd assume it's because the call to the actual `test_X` function ends `prank`?